conf/synapse-coturn.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Coturn
 Documentation=man:coturn(1) man:turnadmin(1) man:turnserver(1)
-After=syslog.target network.target
+After=syslog.target network-online.target
```

Comment:
This is supposed to fix issue #313 but it is not always working.
On my server, after making the change, the synapse-coturn service is still failing to start at boot time.
One way to fix the issue that I have tested and that is working is to add the parameter `-L 0.0.0.0` to the `ExecStart ` line, as in this commit on my fork of the app : https://github.com/Lab-8916100448256/synapse_ynh/commit/d211419b13f40bb1504c0dadf1ee6420304be95d#

Reply:
Well are you really sure about that this parameter `-L 0.0.0.0` will fix the issue ?